src/dhtproto/util/Verify.d

```diff
@@ -0,0 +1,80 @@
+/*******************************************************************************
+
+    Helper module to use `verify` from ocean v3.4 instead of `assert`, even if
+    the ocean submodule is at a version that does not include
+    `ocean.core.Verify`. This is to allow for writing new dhtproto code that can
+    be safely merged into v14.
+
+    Copyright: Copyright (c) 2018 sociomantic labs GmbH. All rights reserved
+
+    License:
+        Boost Software License Version 1.0. See LICENSE.txt for details.
+
+*******************************************************************************/
+
+module dhtproto.util.Verify;
```

Comment:
If it requires adding a whole module as a workaround, I wonder if it might be preferable to just target this feature against dhtproto 14.x.x.

Reply:
It's likely this will be required by any changes to v13.x.x, from now on, so adding it isn't just for the benefit of this specific PR.